lib/README

```diff
@@ -0,0 +1,2 @@
+Unfortunately as of postgis-jdbc 2.1.7, it's not yet available on any central maven repo.
+Supplied here to allow compilation
```

Comment:
It's been a long-time, recurring issue, so think it'll be an uphill battle. Some of it has to do with their build system.  From what I've found above I think the solution is to offer a better alternative through JTS, as just now it appears to be a layer of indirection that's not really required.